sdk/lib/opentelemetry/sdk/trace/span.rb

```diff
@@ -167,20 +167,23 @@ def record_exception(exception, attributes: nil)
 
         # Sets the Status to the Span
         #
-        # If used, this will override the default Span status. Default is OK.
+        # If used, this will override the default Span status. Default has code = Status::UNSET.
```

Comment:
```suggestion
        # If used, this will override the default Span status. Default is UNSET.
```

Reply:
🤷 I was being pedantic - technically, `UNSET` is the `status.code`, not the `status`.

Reply:
Pedantic is permitted :)